test/functional/feature_pak.py

```diff
@@ -329,9 +334,6 @@ def compare(actual, expected):
 
         assert_raises_rpc_error(-8, "bitcoin_descriptor must be a ranged descriptor.", self.nodes[i_pak1].initpegoutwallet, "pkh(tpubD6NzVbkrYhZ4WaWSyoBvQwbpLkojyoTZPRsgXELWz3Popb3qkjcJyJUGLnL4qHHoQvao8ESaAstxYSnhyswJ76uZPStJRJCTKvosUCJZL5B)")
 
-        # key origins aren't supported in 0.17
```

Comment:
nit: we should add "positive" tests since they're supported now... not sure what it'll do